src/php/Lookup/UserAgent.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace CanddiAi\Lookup;
+
+use GuzzleHttp\Client as Client;
+use GuzzleHttp\Exception\RequestException as RequestException;
+use CanddiAi\Singleton\InterfaceSingleton;
+use CanddiAi\Lookup\Response\UserAgent as ResponseUserAgent;
+use CanddiAi\Traits\TraitSingleton;
+
+
+class UserAgent
+    implements InterfaceSingleton
+{
+    use TraitSingleton;
+
+    const c_URL_Agent = 'lookup/useragent/%s';
+
+    public function lookupAgent(
+        $strUserAgent,
+        $strAccountUrl = null,
+        $guidContactId = null
+    )
+    {
+        $strURL = sprintf(self::c_URL_Agent, $strUserAgent);
```

Comment:
@george-hm needs to url encode UserAgent
What if UA was Mozilla/1.1 - would break